.github/workflows/validate-pr.yml

```diff
@@ -75,3 +75,55 @@ jobs:
 
       - name: Jest test
         run: yarn test --passWithNoTests
+
+    lint:
+      name: Run Eslint
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v2
+
+        - name: Setup Node.js
+          uses: actions/setup-node@v2
+          with:
+            node-version: 12
+
+        - name: Cache dependencies
+          id: yarn-cache
+          uses: actions/cache@v2
+          with:
+            path: '**/node_modules'
+            key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
+
+        - name: Install dependencies
+          if: steps.yarn-cache.outputs.cache-hit != 'true'
+          run: yarn install --frozen-lockfile
+
+        - name: Run Eslint
+          run: eslint src/
+
+    prettier:
+      name: Run Prettier
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v2
+
+        - name: Setup Node.js
+          uses: actions/setup-node@v2
+          with:
+            node-version: 12
+
+        - name: Cache dependencies
+          id: yarn-cache
+          uses: actions/cache@v2
+          with:
+            path: '**/node_modules'
+            key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
+
+        - name: Install dependencies
+          if: steps.yarn-cache.outputs.cache-hit != 'true'
+          run: yarn install --frozen-lockfile
+
+        - name: Run Prettier
+          run: prettier --check
```

Comment:
Do we have eslint setup with prettier rules? I _think_ the eslint step above may already handle this. Would be good to double check. Thoughts?

Reply:
Let me double check!